addons/website/views/website_views.xml

```diff
@@ -162,11 +162,38 @@
         </record>
 
         <!-- website.menu views -->
-        <record id="action_website_menu" model="ir.actions.act_window">
-            <field name="name">Website Menu</field>
-            <field name="res_model">website.menu</field>
-            <field name="view_mode">list</field>
-            <field name="context">{'search_default_my_websites':1}</field>
```

Comment:
add back, magic word

Reply:
Too late for v11, please rebuild on master and with the come back of ' magic word' and ok to merge it